src/tensors/levels/sparserlelevels.jl

```diff
@@ -233,12 +237,12 @@ function instantiate_reader(fbr::VirtualSubFiber{VirtualSparseRLELevel}, ctx, ::
                                 body = (ctx, ext) -> Thunk( 
                                      body = (ctx) -> Pipeline([
                                         Phase(
-                                            stop = (ctx, ext) -> call(-, value(my_i_start), 1),
+                                            stop = (ctx, ext) -> call(-, value(my_i_start), is_continuous_extent(ext) ? Eps : Ti(1)),
```

Comment:
Can you make this a function like `get_after(ext)` which specializes on `ContinuousExtent`? The boolean looks not very extensible.

Reply:
or perhaps `get_eps(ext::Extent) = literal(1)` and `get_eps(ext::ContinuousExtent) = Eps`?